### ISSUE TITLE
Calls Online

### ISSUE THREAD

**Issue by xceedconnections** [magnussolution/magnusbilling#568]
It will be really good to see Callerid in Calls online page..
it will help alot..
right now we need to see which callerid is used in the  CDR menu.. please add this feature in calls online too..

thanks.

**Comment by magnussolution** [comment]
hi, clique on the call on call online, you will see the callerid on the form

**Comment by xceedconnections** [comment]
Yes, but can that be added in the main page after or before the destination
number.
